#Info retrieved from a tag M6e during an inventory
#
# ID         tag EPC code
# TagStats   additional physical data (time, power, phase, freq) from tag readings

string     ID
TagStats[] stats

# NOTE : WE ARE NOT READING TAG data!!
